assets/constants.ts

```diff
@@ -5899,6 +5899,11 @@ export default {
   // page.
   "OPPORTUNITIES_PAGE_SIZE": 10,
 
+  // The breakpoint for mobile view for contributor dashboard in px.
+  // This value must be the same as the one specified in
+  // opportunities-list-item.component.html.
+  "CONTRIBUTOR_DASHBOARD_MOBILE_BREAKPOINT": 700,
```

Comment:
Will this only be relevant only for opportunities-list-item.component.html? If so, then let's update the constant name to reflect that, since currently it makes it sound like the breakpoint value is relevant for the entire contributor dashboard, which I don't think it is (e.g. I remember there being many different breakpoint values in the last PR).

Reply:
Fixed.